api/comments/serializers.py

```diff
@@ -111,6 +113,8 @@ def update(self, comment, validated_data):
                     comment.edit(content, auth=auth, save=True)
                 except PermissionsError:
                     raise PermissionDenied('Not authorized to edit this comment.')
+                except ValidationValueError:
+                    raise ValidationError('Ensure this field has no more than 500 characters.')
```

Comment:
It is a little weird because the max_length validator was previously on the [serializer](https://github.com/CenterForOpenScience/osf.io/blob/develop/api/comments/serializers.py#L40) and now the length check is only on the [model](https://github.com/leb2dg/osf.io/blob/f59b9b72886868f16c993571fe99229484a56239/website/project/model.py#L233) since the content needs to be processed first. So the model is throwing the modularodm `ValidationValueError` and the serializer is catching it and raising the rest framework `ValidationError` it would normally throw.